docs/baremetalhost-states.md

```diff
@@ -11,10 +11,10 @@ Newly created hosts move immediately to Discovered or Registering. No
 host stays in the Created state while the operator is working
 properly.
 
-## Discovered
+## Unmanaged
 
-A Discovered host is missing either the BMC address or credentials
-secret name, and does not have enough information to access the BMC
+An Unmanaged host is missing both the BMC address and credentials
```

Comment:
Does it need to be both? Could it be either?

Reply:
Working assumption:
* If you didn't provide either, you meant to create an unmanaged host
* If you provided one but not the other, linked to a missing credential secret, or linked to an incorrectly formatted secret, you screwed up and should see big error messages in the UI.

Reply:
Come to think of it, it's academic anyway because both are required fields within the BMC struct. So you can only specify both or neither.